doc/pipeline.rst

```diff
@@ -821,10 +821,32 @@ combined:
   deletions (0 copies) (``ampdel``).
 - Confidence interval overlapping zero (``ci``).
 - Standard error of the mean (``sem``), a parametric estimate of confidence
-  intervals which behaves similarly.
+  intervals which behaves *similarly*.
 
 In each case, adjacent segments with the same value according to the given
 criteria are merged together and the column values are recalculated
 appropriately. Segments on different chromosomes or with different
 allele-specific copy number values will not be merged, even if the total copy
 number is the same.
+
+
+.. _bintest:
+
+``bintest``
+-----------
```

Comment:
Second, regarding the question you raised about the formats

> I think there is a confusion between:
>
> * Extension of the file produced through batch ".bintest.cns"
> * Output format produced by bintest, which is actually closer to a ".cnr" (regarding coordinates mostly)

I think this is a really good point, and the fact that the file is currently called `.bintest.cns` is probably a typo. Let's rename it to `.bintest.cnr` as part of this PR as well (as well as any occurrences of `.bintest.cns` in the code which you might find)

Reply:
Looking at this line from `batch.py`:
https://github.com/etal/cnvkit/blob/25d91cdb962639c98d041c3b05c7de659c41dc70/cnvlib/batch.py#L210
=> I am not sure if this is really a typo
=> Because conceptually, results produced are "called CNV" so closer to a ".cns" than to a ".cnr", where no CNV calling is made yet (even if this does not match actual file format)
=> Plus this is done late into calling pipeline, so it would probably be distrurbing for a user to have this second ".bintest.cnr" ?

Maybe a solution to improve consistency would be to **actually write a ".cns", but with a dummy "probes" column always at "1"**? (regarding [doc about ".cns"](https://cnvkit.readthedocs.io/en/stable/fileformats.html#segmented-log2-ratios-cns))

Reply:
The default "probes=1" is a good idea.

The reason this is .cns rather than .cnr is that these are meant to represent copy number calls rather than input data. If `scatter` is extended to allow multiple .cns files as input, then these bin-level calls and the coarse-grained segmentation could be plotted together and would appear as bright colors (a colormap starting with darkorange) rather than as gray raw data points.